content/pt-br/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Sidecar Containers
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.29" state="beta" >}}
+
+Os sidecar containers são contêineres secundários que rodam junto (lado a lado) com o contêiner da aplicação principal dentro do mesmo {{< glossary_tooltip text="Pod" term_id="pod" >}}.
```

Comment:
Instead of using the original "sidecar containers" you change to something more native, like:
* contêineres sidecar

Or something like that, please check all the other occurrences.